mobile/examples/object_detection/android/README.md

```diff
@@ -0,0 +1,44 @@
+# ONNX Runtime Mobile Object Detection Android sample application with Ort-Extensions support for pre/post processing
+
+## Overview
+
+This is a basic Object Detection sample application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android with [Ort-Extensions](https://github.com/microsoft/onnxruntime-extensions) support for pre/post processing. The demo app accomplishes the task of detecting objects from a given image.
+
+The model used here is from source: [Yolov8 in extensions](https://github.com/microsoft/onnxruntime-extensions/blob/64f20828ce0291394886e277c23529cd1d11320d/tutorials/yolo_e2e.py#L37) and with pre/post processing support.
+
+This model (Yolov8n) can be fed with image bytes directly and outputs the detected objects with bounding boxes. Please note that scores and labels are not drawn by this model. Boxes in different classes are drawn with different colors.
```

Comment:
Understand that for now, we only support drawing bounding boxes on the detected objects. 

wondering if the model can at least output the detected objects name lists or scores, (like for example, we can print to console log or anything? don't have to draw to the picture.)

Reply:
Drawing classes and scores as well